packages/xo-server/src/api/backup-ng.js

```diff
@@ -197,6 +197,16 @@ getLogs.params = {
   '*': { type: 'any' },
 }
 
+export function clearLogsCache() {
+  return this.clearBackupNgLogs()
+}
+
+clearLogsCache.permission = 'admin'
+
+clearLogsCache.params = {
+  runId: { type: 'string', optional: true },
+}
```

Comment:
Avoid adding a new method for this, simply add a `_forceRefresh` parameter to `backupNg.getLogs`.

Please add a comment explaining this is a temporary work-around that will disappear when the log consolidation is done.